app/assets/stylesheets/base/_mixins.scss

```diff
@@ -32,3 +32,12 @@
 		font-style: $style;
 	}
 }
+@mixin clear-fix() {
+	&:after {
+	   content: " "; /* Older browser do not support empty content */
```

Comment:
Use `//` comments everywhere<br>Properties should be ordered clear, content, display, height, visibility